migrations/20240510231712_/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE `post` MODIFY `content` TEXT NULL;
```

Comment:
We've had issues with this in the past, but `post` should be capitalized here to ensure we're altering the correct table.